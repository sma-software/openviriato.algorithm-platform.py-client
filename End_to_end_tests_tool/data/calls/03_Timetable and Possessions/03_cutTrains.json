{
    "restResource": "/trains",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "FromTime",
            "value": "2003-05-01T00%3A10%3A00",
            "optional": false
        },
        {
            "key": "ToTime",
            "value": "2003-05-01T00%3A11%3A30",
            "optional": false
        },
        {
            "key": "CutTrain",
            "value": "true",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetTrainsCutToTimeRange(timeWindow)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.ITimeWindow" ],
    "subcategory": "Get Trains"
}
