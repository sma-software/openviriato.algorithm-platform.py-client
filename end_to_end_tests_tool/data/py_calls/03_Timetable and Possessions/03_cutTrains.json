{
  "PythonParameterFromJson" :
  [
    {
      "ParameterName": "time_window",
      "aidm_factory": "convert_json_to_time_window",
      "ObjectJson": {
        "from_time": "2003-05-01T00:10:00",
        "to_time": "2003-05-01T00:11:30"
      }
    }
  ]
}