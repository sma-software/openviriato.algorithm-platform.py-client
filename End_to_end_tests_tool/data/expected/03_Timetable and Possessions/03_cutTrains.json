[{"ID":3516,"TrainPathNodes":[{"ID":2424,"SectionTrackID":null,"NodeID":7,"NodeTrackID":8,"FormationID":1187,"ArrivalTime":"2003-05-01T00:10:00","DepartureTime":"2003-05-01T00:10:00","MinimumRunTime":null,"MinimumStopTime":"P0D","StopStatus":"CommercialStop","SequenceNumber":0},{"ID":2788,"SectionTrackID":1175,"NodeID":15,"NodeTrackID":16,"FormationID":1187,"ArrivalTime":"2003-05-01T00:14:00","DepartureTime":"2003-05-01T00:14:00","MinimumRunTime":"PT4M","MinimumStopTime":"P0D","StopStatus":"Passing","SequenceNumber":1}],"DebugString":"RVZH_3_1_J03 tt_(S)"},{"ID":2060,"TrainPathNodes":[{"ID":1696,"SectionTrackID":null,"NodeID":10,"NodeTrackID":null,"FormationID":null,"ArrivalTime":"2003-05-01T00:10:00","DepartureTime":"2003-05-01T00:10:00","MinimumRunTime":null,"MinimumStopTime":"P0D","StopStatus":"CommercialStop","SequenceNumber":1}],"DebugString":"RVZH_1_1_J03 tt_(G)"}]